### PR TITLE
Merged UI and page Stats because it wasn't done before

### PR DESCRIPTION
**Pull request by Naludrag**
working on #20 
The admin stats is not done yet but stats for questions and debates are done

### CODE REVIEW

File: src/app/pages/admin/new-debate/new-debate.page.scss [hunk 1]
```diff
@@ -1,22 +1,76 @@
 ion-content ion-toolbar {
-    --background: translucent;
+  --background: translucent;
+}
+ion-toolbar{
+  --color:white;
+  //--background:rgb(9, 12, 54);
```
Comment (BecaudA) [inline src/app/pages/admin/new-debate/new-debate.page.scss:6]: Remove unused code or specify why you keeped it in the code.

File: src/app/pages/admin/new-debate/new-debate.page.scss [hunk 2]
```diff
@@ -1,22 +1,76 @@
...
+  --background: #006494; 
+}
+
+ion-content{
+color: rgb(9, 12, 54) !important;
+font-size: 18px;
+text-align: center;
+--ion-background-color: white;//#adddf1;
```
Comment (BecaudA) [inline src/app/pages/admin/new-debate/new-debate.page.scss:14]: Remove comment or specify why you keeped it in the code.

File: src/app/pages/admin/new-debate/new-debate.page.scss [hunk 3]
```diff
@@ -1,22 +1,76 @@
...
-    left: calc(50% - 70px);
+  p{
+    margin-left: 15%;
+    margin-right: 15%;
+    font-family:"Gill Sans", sans-serif;
+
   }
+@media(min-width:800px){
+  .logo{
+  //  width:80%;
```
Comment (BecaudA) [inline src/app/pages/admin/new-debate/new-debate.page.scss:65]: Remove unused code or specify why you keeped it in the code.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
+<ion-header [translucent]="true">
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button defaultHref="debate/questions"></ion-back-button>
+    </ion-buttons>
+    <ion-title>
+      Stats of the question {{questionManager.getSavedQuestion().title}}
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.html:7]: Traduire en français.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.html [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      <ion-label>
+        <ion-text color="primary">
+          <h1>Réponse</h1>
+        </ion-text>
+      </ion-label>
+    </ion-item>
+    <ion-item *ngFor="let response of statResponses[2]"  class="response-item">
+      <ion-label>{{response.response}}</ion-label>
+    </ion-item>
+    <strong *ngIf="statResponses.length != 3" class="wait-text">No stats to be shown for the moment</strong>
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.html:48]: Traduire en français.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
+
+ion-toolbar{
+  --color:white;
+  //--background:rgb(9, 12, 54);
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.scss:4]: Remove unused code or specify why you keeped it in the code.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.scss [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+}
+
+
+#container p {
+  font-size: 16px;
+  line-height: 22px;
+
+  color: #006494; //#8c8c8c;
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.scss:29]: Remove comment or specify why you keeped it in the code.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+      private statManager     : StatService,
+      private questionManager : QuestionService
+  ) {}
+
+  /**
+   * Updates the list of questions
+   */
+  private async getStats() {
+    console.log("Sending things")
+    console.log(this.questionId)
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts:46]: Missing `;` at the end of instructions.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+              }
+            }
+          ]
+        }
+      }
+    });
+    this.barChart.data.datasets.push({
+      label: "# number of votes",
+      data: []
+      //color: if we need a color
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts:75]: Remove unused code or specify why you keeped it in the code.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+    });
+    for (const response of this.statResponses[2]) {
+      console.log(response); // To Remove
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts:78]: Add a TODO or remove  it now.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+    this.doughnutChart = new Chart(this.doughnutCanvas.nativeElement, {
+      type: "doughnut",
+      data: {
+          datasets: []
+      }
+    });
+    this.doughnutChart.data.datasets.push({
+      label: "# Percentage of votes",
+      data: []
+      //color: if we need a color
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts:100]: Remove unused code or specify why you keeped it in the code.

File: src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+    });
+    for (const response of this.statResponses[2]) {
+      console.log(response); // To Remove
```
Comment (BecaudA) [inline src/app/pages/admin/openquestion-stats/openquestion-stats.page.ts:103]: Add  a TODO or remove it.

File: src/app/pages/client/help/help.page.scss [hunk 13]
```diff
@@ -0,0 +1,76 @@
...
+        margin-left: 120px;
+        margin-right:120px;
+    }
+  }
+
+  
+
+
+
+
```
Comment (BecaudA) [inline src/app/pages/client/help/help.page.scss:76]: Remove empty line.

File: src/app/pages/client/login/login.page.scss [hunk 14]
```diff
@@ -1,18 +1,97 @@
+ion-content {
+    --ion-background-color:rgb(153, 204, 228);
+    --color: white     ;//B4B1D3;
```
Comment (BecaudA) [inline src/app/pages/client/login/login.page.scss:3]: Remove comment or specify why you keeped it in the code.

File: src/app/pages/client/questions/questions.page.html [hunk 15]
```diff
@@ -8,12 +11,29 @@
...
+        <img src="assets/icon/Groupe 14.png"  class = "logo1">
+        <img src="assets/icon/Groupe 26.png"  class = "logo1">
+      </div><br>
+    </ion-card>
+      <p class="other">Préparez-vous à réagir au débat et à poser vos propres questions en direct</p>
+  </div>
+
+  <div id="container" class="second">
+    <strong *ngIf="availableQuestions.length == 0" class="wait-text">
+      <p>En attente de questions </p>
```
Comment (BecaudA) [inline src/app/pages/client/questions/questions.page.html:28]: Espace inutile à supprimer `...questions </p>`

File: src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.html [hunk 16]
```diff
@@ -1,10 +1,10 @@
 <ion-header [translucent]="true">
   <ion-toolbar>
     <ion-buttons slot="start">
-      <ion-back-button defaultHref="admin-home"></ion-back-button>
+      <ion-back-button defaultHref="debate/questions"></ion-back-button>
```
[the remark this answers is not in the document]
Comment (Naludrag) [reply src/app/pages/admin/closedquestion-stats/closedquestion-stats.page.html:4]: C'est parce que le page pour revenir aux question s'appellent debate/questions

**Pull request merged**